src/main/java/ru/yanmayak/TelegramNotifyBot/bot/TelegramBot.java

```diff
@@ -0,0 +1,45 @@
+package ru.yanmayak.TelegramNotifyBot.bot;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.telegram.telegrambots.bots.TelegramLongPollingBot;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.objects.Update;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+
+@Component
+public class TelegramBot extends TelegramLongPollingBot {
+
+    @Value("${bot.token}")
```

Comment:
annotation should be used in constructor, othervise it will be complicated to test your class